### PR TITLE
Additional bindings for verification keys

### PR DESCRIPTION
**Pull request by imeckler**
This PR adds a few C  bindings for verification keys' fields (on the C side).

### CODE REVIEW

File: src/lib/snarky/src/camlsnark_c/libsnark-caml/libsnark/caml/caml_mnt4.cpp [hunk 1]
```diff
@@ -632,6 +632,29 @@ r1cs_gg_ppzksnark_verification_key<ppT>* camlsnark_mnt4_verification_key_of_stri
   return vk;
 }
 
+std::vector<libff::G1<ppT>>*
+camlsnark_mnt4_verification_key_query(r1cs_gg_ppzksnark_verification_key<ppT>* vk) {
+  std::vector<libff::G1<ppT>>* res = new std::vector<libff::G1<ppT>>();
+  printf("Test\n");
```
Comment (bkase) [inline src/lib/snarky/src/camlsnark_c/libsnark-caml/libsnark/caml/caml_mnt4.cpp]: delete this

File: src/lib/snarky/src/camlsnark_c/libsnark-caml/libsnark/caml/caml_mnt4.cpp [hunk 2]
```diff
@@ -632,6 +632,29 @@ r1cs_gg_ppzksnark_verification_key<ppT>* camlsnark_mnt4_verification_key_of_stri
...
+  res->emplace_back(vk->ABC_g1.first);
+  for (size_t i = 0; i < vk->ABC_g1.rest.values.size(); ++i)
+  {
+  printf("Test %d\n", i);
```
Comment (bkase) [inline src/lib/snarky/src/camlsnark_c/libsnark-caml/libsnark/caml/caml_mnt4.cpp]: and this

File: src/lib/snarky/src/camlsnark_c/libsnark-caml/libsnark/caml/caml_mnt6.cpp [hunk 3]
```diff
@@ -632,6 +632,29 @@ r1cs_gg_ppzksnark_verification_key<ppT>* camlsnark_mnt6_verification_key_of_stri
...
 }
 
+std::vector<libff::G1<ppT>>*
+camlsnark_mnt6_verification_key_query(r1cs_gg_ppzksnark_verification_key<ppT>* vk) {
+  std::vector<libff::G1<ppT>>* res = new std::vector<libff::G1<ppT>>();
+  printf("Test\n");
+  res->emplace_back(vk->ABC_g1.first);
+  for (size_t i = 0; i < vk->ABC_g1.rest.values.size(); ++i)
+  {
+  printf("Test %d\n", i);
```
Comment (bkase) [inline src/lib/snarky/src/camlsnark_c/libsnark-caml/libsnark/caml/caml_mnt6.cpp]: delete this

Comment (imeckler) [reply src/lib/snarky/src/camlsnark_c/libsnark-caml/libsnark/caml/caml_mnt6.cpp]: whoops -- thank you

**Pull request merged**